cumulus/client/cirrus-executor/src/processor.rs

```diff
@@ -216,6 +216,12 @@ where
 		let execution_receipt =
 			ExecutionReceipt { primary_hash, secondary_hash: header_hash, trace, trace_root };
 
+		crate::aux_schema::write_execution_receipt::<_, Block>(
+			&*self.client,
+			header_hash,
+			&execution_receipt,
+		)?;
```

Comment:
TODO is needed that this may be never cleaned up (as we discussed in DMs)

Reply:
https://github.com/subspace/subspace/pull/236/commits/2587e806d8ce5506352d49f39a229c5196d302c6 Please see if it could serve as a starting point.